engine/src/cubos/engine/input/input.cpp

```diff
@@ -0,0 +1,69 @@
+#include <cubos/core/log.hpp>
+
+#include <cubos/engine/input/input.hpp>
+
+using namespace cubos::engine;
+
+void Input::clear()
+{
+    // TODO: Deregister from all events
+    mBindings.clear();
+    CUBOS_DEBUG("Input cleared");
+}
+
+void Input::clear(int player)
+{
+    // TODO: Deregister from all the events for this player
+    mBindings.erase(player);
+    CUBOS_DEBUG("Input cleared for player {}", player);
+}
+
+void Input::setBindings(int player, const Bindings& bindings)
+{
+    // TODO: Register to all the events for this player
+    mBindings[player] = std::move(bindings);
```

Comment:
The move doesn't do anything here you can't move a type behind a `const` reference - you can remove it.